src/content/developers/docs/nodes-and-clients/run-a-node/index.md

```diff
@@ -97,28 +104,69 @@ Before starting Ethereum client software, perform a last check that your environ
 - System has correct time and date.
 - Your router and firewall accept connections on listening ports. By default Ethereum clients use a listener (TCP) port and a discovery (UDP) port, both on 30303 by default.
 
-Run your client on a testnet first to help make sure everything is working correctly. [Running a Geth light node](/developers/tutorials/run-light-node-geth/) should help.
-You need to declare any client settings that aren't default at the start. You can use flags or the config file to declare your preferred configuration. Check out your client's documentation for the specifics
+Run your client on a testnet first to help make sure everything is working correctly. You need to declare any client settings that aren't default at the start.
+In order to connect to a consensus client the execution client must generate a `jwtsecret` at a known path. This path must be known by both clients as it is used
```

Comment:
Still trying to find an answer to this, but do we know if this is safe to set up (on Mainnet at least) before the Bellatrix/Paris client updates?

Reply:
yea maybe good to add a clear warning to practise this on a testnet for now and wait for an announcement that the merge releases are available before doing it on Mainnet.